index.html

```diff
@@ -44,6 +44,187 @@ <h3 class="headline-intro">
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
+          <ul class="tech-stack-2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
+          <ul class="tech-stack-2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+      </div>
+    </div>
+  <!--Recent Works Section Ends-->
+  <!--About Me Section Begins-->
+    <div class="about-me-container">
+      <div class="about">
+        <h6>About Me</h6>
+        <div class="about-content">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </div>
+        <button type="button">Get My Resume</button>
+      </div>
+      <hr class="about-divider">
+      <div class="skills">
+        <div class="languages">
+          <div class="list-title">
+            <img src="./images/Rectangle_55.svg" alt="">
+            <h6>Languages</h6>
+          </div>
+        <ul class="list-container">
+          <li>Javascript</li>
+          <li>html</li>
+          <li>Ruby</li>
+          <li>css</li>
+        </ul>
+        </div>
+        <div class="frameworks">
+          <div class="list-title">
+            <img src="./images/Rectangle_56.svg" alt="">
+            <h6>Frameworks</h6>
+          </div>
+          <ul class="list-container">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skills">
+          <div class="list-title">
+            <img src="./images/Rectangle_57.svg" alt="">
+            <h6>Skills</h6>
+          </div>
+          <ul class="list-container">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>GitLab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Wrapping the about section in a `section` tag will also make your code readable, understandable and much better because it is the right semantic tag to use. Please make a refactor here.